script/bootstrap

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env bash
 set -e
 
-# Only set GOPATH on non-Windows platforms as Windows can't do the symlinking
-# that we need.
-if [ -z "$GOPATH" ] && uname -s | grep -vq "_NT-"; then
+# Tell MSYS to really create symbolic links.
+export MSYS=winsymlinks:nativestrict
```

Comment:
I guess so, but then calling script/bootstrap on a local Windows developer machine (from Git Bash) would not properly work unless that variable is already set.